include/dem/pw_contact_force.h

```diff
@@ -114,6 +124,14 @@ class PWContactForce
     // Calculation of total force
     Tensor<1, dim> total_force = normal_force + tangential_force;
 
+    if (calculation_force_torque == true)
+      {
+        ForceOnWall[boundary_id] = ForceOnWall[boundary_id] - total_force;
+        TorqueOnWall[boundary_id] =
+          TorqueOnWall[boundary_id] -
```

Comment:
We don't use capital letter for variable names. force_on_walls and torque_on_walls

Reply:
Yes